src/hotspot/share/gc/shared/space.inline.hpp

```diff
@@ -87,7 +87,9 @@ class DeadSpacer : StackObj {
   CompactibleSpace* _space;
 
 public:
-  DeadSpacer(CompactibleSpace* space) : _allowed_deadspace_words(0), _space(space) {
+  size_t _dead_space;
```

Comment:
Should this really be "public"? Maybe `friend`-ing with the only user is better?

Reply:
Yep. Befriended.